### PR TITLE
Draft for the May's Newsletter in the filename: newsletter_14_May2020.md

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* May 2020 Newsletter draft for The Turing Way

### List of changes proposed in this PR (pull-request)

* Content for the May newsletter added
* Images uploaded

### What should a reviewer concentrate their feedback on?

- [ ] Language, typo, content and relevance
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
Newsletter with images can be viewed on this branch: https://github.com/alan-turing-institute/the-turing-way/blob/malvikasharan-newsletter-May/communications/newsletters/newsletter_14_May2020.md

File: communications/newsletters/newsletter_14_May2020.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+# Attend this week’s Collaboration Café, learn about Mental Health Awareness Week and find other updates!
+
+Hello Turing Way friends!
+
+This month as we continue to adjust to the current norm, we wish you good health and hope that you and your loved ones are keeping well. 
+It’s not easy keeping morale up with the current state of the pandemic, therefore, in this newsletter, we talk about the importance of self-care and highlight the Mental Health Foundation who are hosting [Mental Health Awareness Week](https://www.awarenessdays.com/awareness-days-calendar/mental-health-awareness-week-2020/) from 18 to 24 May 2020. 🌻
+
+We congratulate Kirstie Whitaker, the project lead of _The Turing Way_ for starting her new role as the [lead of Tools, Practices and Systems research programme](https://www.turing.ac.uk/news/dr-kirstie-whitaker-named-institutes-programme-lead-open-source-tools-practices-and-systems) at The Alan Turing Institute, London. 🎊
+
+🗓 Join us at our upcoming Collaboration Café this week on 20 May 2020 at 19:00 ([check your local time](https://arewemeetingyet.com/London/2020-05-20/19:00/TuringWay-CollaborationCafe)).
```
Comment (sgibson91) [inline communications/newsletters/newsletter_14_May2020.md:10]:
```suggestion
🗓 Join us at our upcoming Collaboration Café this week on 20 May 2020 at 19:00 BST ([check your local time](https://arewemeetingyet.com/London/2020-05-20/19:00/TuringWay-CollaborationCafe)). 
```

File: communications/newsletters/newsletter_14_May2020.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+It’s not easy keeping morale up with the current state of the pandemic, therefore, in this newsletter, we talk about the importance of self-care and highlight the Mental Health Foundation who are hosting [Mental Health Awareness Week](https://www.awarenessdays.com/awareness-days-calendar/mental-health-awareness-week-2020/) from 18 to 24 May 2020. 🌻
+
+We congratulate Kirstie Whitaker, the project lead of _The Turing Way_ for starting her new role as the [lead of Tools, Practices and Systems research programme](https://www.turing.ac.uk/news/dr-kirstie-whitaker-named-institutes-programme-lead-open-source-tools-practices-and-systems) at The Alan Turing Institute, London. 🎊
+
+🗓 Join us at our upcoming Collaboration Café this week on 20 May 2020 at 19:00 ([check your local time](https://arewemeetingyet.com/London/2020-05-20/19:00/TuringWay-CollaborationCafe)). 
+More details are available on [this HackMD](https://hackmd.io/@KirstieJane/CollabCafe).
+
+📢 [International Neuroinformatics Coordinating Facility (INCF)](https://www.incf.org/blog/call-mentors-join-us-gsod-2020) is participating at the [Google Seasons of Docs](https://developers.google.com/season-of-docs/docs/participants).
```
Comment (sgibson91) [inline communications/newsletters/newsletter_14_May2020.md:13]:
```suggestion
📢 [International Neuroinformatics Coordinating Facility (INCF)](https://www.incf.org/blog/call-mentors-join-us-gsod-2020) is participating in the [Google Seasons of Docs](https://developers.google.com/season-of-docs/docs/participants).
```

File: communications/newsletters/newsletter_14_May2020.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+![Tweet about Kirstie's new position](images/kw-May2020.png)
+
+*Tweet by [The Turing Institute](https://twitter.com/turinginst/status/1255054249807024128?s=20)*
+
+## Upcoming workshops and events
+
+### Boost Research Reproducibility with Binder Workshop
+
+In collaboration with [UK DRI](https://ukdri.ac.uk/), _The Turing Way_ is hosting a half-day workshop on Binder on 11 June 2020, from 13:00 to 17:00.
```
Comment (sgibson91) [inline communications/newsletters/newsletter_14_May2020.md:49]:
```suggestion
In collaboration with [UK DRI](https://ukdri.ac.uk/), _The Turing Way_ is hosting a half-day workshop on Binder on 11 June 2020, from 13:00 to 17:00 BST.
```

**Comment by malvikasharan** [comment]
Hi @crangelsmith, I am tagging you to check if you would have time to give this newsletter draft a review. If you have less time, and won't be able to review the newsletter, can you please approve the section about you?

File: communications/newsletters/newsletter_14_May2020.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+Binder allows you to share your software from GitHub in a single clickable link, making it easier for users to not worry about the computational environments, dependencies and platforms.
+In this workshop, we will discuss reproducible computing environments, show examples of others’ projects in myBinder.org and help you learn how to prepare a binder-ready project. 
+At the end of the workshop, you will be able to take some of your own content (in Jupyter notebook or scripts) and prepare it so that it can be used by others on myBinder.org.
+More information is available on the [registration page online](https://www.eventbrite.co.uk/e/workshop-boost-research-reproducibility-with-binder-registration-105228216428).
+If you have previous experience with Binder and would like to join as a helper, please reach out to us by emailing theturingway@gmail.com.
```
Comment (sgibson91) [inline communications/newsletters/newsletter_14_May2020.md:54]:
```suggestion
If you have previous experience with Binder and would like to join as a helper, please reach out to us by emailing [theturingway@gmail.com](mailto:theturingway@gmail.com).
```

File: communications/newsletters/newsletter_14_May2020.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+### Google Seasons of Docs
+
+INCF is taking part in this year's [Google Seasons of Docs](https://developers.google.com/season-of-docs/docs/participants) as an open source organisation and _The Turing Way_ is listed as one of its projects.
```
Comment (sgibson91) [inline communications/newsletters/newsletter_14_May2020.md:58]:
```suggestion
INCF is taking part in this year's [Google Seasons of Docs](https://developers.google.com/season-of-docs/docs/participants) (GSOD) as an open source organisation and _The Turing Way_ is listed as one of its projects.
```

Introducing the acronym you use below

File: communications/newsletters/newsletter_14_May2020.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+We have compiled some recommendations that can be used as the starting points for the GSOD candidates.
+Please find details on [this issue](https://github.com/alan-turing-institute/the-turing-way/issues/1075), and feel free to comment there if any further information will be useful.
+
+### Data Feminism Book and “Challenging Power in Data Science” seminar 
+
+[Data Feminism](https://mitpress.mit.edu/books/data-feminism) is a recent book by Catherine D'Ignazio and Lauren F. Klein that explains the concept of data feminism as a way of thinking about data science and its communication informed by decades of intersectional feminist activism and critical thought. 
+They will be giving a talk hosted by The Alan Turing Institute to discuss how challenges to the male/female binary can challenge other hierarchical classification systems; how an understanding of emotion can expand our ideas about effective data visualization; and how the concept of “invisible labor” can expose the significant human efforts required by our automated systems.
```
Comment (sgibson91) [inline communications/newsletters/newsletter_14_May2020.md:68]: You use challenges twice in this sentence, is there another word we could use?

File: communications/newsletters/newsletter_14_May2020.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+To avoid feeling disconnected while you work from home, you can join others over online coffee breaks or co-working calls. 
+These are short informal calls that can help you stay connected with others in your workplace and social network, create opportunities to exchange ideas, stay motivated and show kindness towards each other. 
+Kirstie has shared her thoughts and ideas on running effective virtual coffee calls on [this issue](https://github.com/alan-turing-institute/the-turing-way/issues/965).
+
+### Share your tips on self-care
+
+We would love to hear your tips and recommendations on self-care and ensuring the well being of your team members. Please comment under [this issue](https://github.com/alan-turing-institute/the-turing-way/issues/965) on the GitHub repository.
+Your recommendations will be included to the chapter on remote collaboration ([see details](https://github.com/alan-turing-institute/the-turing-way/pull/962)).
+
+Have suggestions to improve accessibility in our community? Reach out to us by emailing theturingway@gmail.com.
```
Comment (sgibson91) [inline communications/newsletters/newsletter_14_May2020.md:135]:
```suggestion
Have suggestions to improve accessibility in our community? Reach out to us by emailing [theturingway@gmail.com](mailto:theturingway@gmail.com).
```

File: communications/newsletters/newsletter_14_May2020.md (hunk 6 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_14_May2020.md:68]:
```suggestion
They will be giving a talk hosted by The Alan Turing Institute to discuss how challenging the male/female binary system can help investigate other hierarchical classification systems; how an understanding of emotion can expand our ideas about effective data visualization; and how the concept of “invisible labor” can expose the significant human efforts required by our automated systems. 
```

Comment (malvikasharan) [reply communications/newsletters/newsletter_14_May2020.md:68]: I took that from the eventbrite description, rephrased that now.

**Comment by malvikasharan** [comment]
@sgibson91 Thank you so much for such a quick response and correcting the errors. 💖🎉

File: communications/newsletters/newsletter_14_May2020.md [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+See [the workshop page](https://n8cir.org.uk/events/remote-reprohack/) for details. 
+
+## Tips & Tricks for new contributors
+
+Are you new to _The Turing Way_ and would like to learn more about the project? Listen to the insightful talk delivered by the project lead, Kirstie Whitaker, at [OpenMR Benelux 2020](https://openmrbenelux.github.io/products/product08/) recording from which is available [online on YouTube](https://www.youtube.com/watch?v=dlycvMU45ek&feature=youtu.be).
+
+### Contributor in focus: Camila Rangel Smith
+
+Camila Rangel Smith is a Research Data Scientist at The Alan Turing Institute.
+She is also a co-founder of the [CEVALE2VE project](http://www.cevale2ve.org/en/home/) that has become consolidated into [LA-CoNGA physics](http://laconga.redclara.net/), an EU Erasmus+ funded project with a mission to create a Latin American and European Community for Advanced Physics.
```
Comment (crangelsmith) [inline communications/newsletters/newsletter_14_May2020.md:96]: Could we put a capital P in Physics? Its my fault that i didn't write it correctly on the email :)

File: communications/newsletters/newsletter_14_May2020.md [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+She is translating _The Turing Way_ resources into Spanish in collaboration with her friends and colleagues from Venezuela and Mexico.
```
Comment (crangelsmith) [inline communications/newsletters/newsletter_14_May2020.md:97]: I think the contributors are from Venezuela and Spain, I'm not aware of anyone in Mexico at the moment. :)

File: communications/newsletters/newsletter_14_May2020.md [hunk 10]
```diff
@@ -0,0 +1,192 @@
...
+
+### Contributor in focus: Camila Rangel Smith
+
+Camila Rangel Smith is a Research Data Scientist at The Alan Turing Institute.
+She is also a co-founder of the [CEVALE2VE project](http://www.cevale2ve.org/en/home/) that has become consolidated into [LA-CoNGA physics](http://laconga.redclara.net/), an EU Erasmus+ funded project with a mission to create a Latin American and European Community for Advanced Physics.
+She is translating _The Turing Way_ resources into Spanish in collaboration with her friends and colleagues from Venezuela and Mexico. 
+In her words: *“I think The Turing Way is an excellent resource that can change the way science is done (I wish I had it when I started my PhD). 
+Although the international language of science is English, I know for a fact that not everyone in places like Latin-American has the time and resources to learn it, so I think we must do everything we can to break those barriers and improve the accessibility of knowledge for everyone.
```
Comment (crangelsmith) [inline communications/newsletters/newsletter_14_May2020.md:99]: My fault again, I think is "Latin-America" without the "n" at the end.

**Comment by HeidiSeibold** [comment]
@malvikasharan any specific reason you requested my review?

I went through the newsletter and it looks good to me :clap:

File: communications/newsletters/newsletter_14_May2020.md (hunk 9 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_14_May2020.md:97]: Ah! Thanks for pointing that. I am correcting that right away.

**Comment by crangelsmith** [comment]
> Hi @crangelsmith, I am tagging you to check if you would have time to give this newsletter draft a review. If you have less time, and won't be able to review the newsletter, can you please approve the section about you?

The newsletter looks great! I just added a couple of comments on typos and minor corrections. 
Thanks again @malvikasharan for including me :)

File: communications/newsletters/newsletter_14_May2020.md (hunk 9 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_14_May2020.md:97]: Ah! Thanks for pointing that. I am correcting that right away.

File: communications/newsletters/newsletter_14_May2020.md (hunk 10 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_14_May2020.md:99]:
```suggestion
Although the international language of science is English, I know for a fact that not everyone in places like Latin-America has the time and resources to learn it, so I think we must do everything we can to break those barriers and improve the accessibility of knowledge for everyone. 
```

File: communications/newsletters/newsletter_14_May2020.md (hunk 8 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_14_May2020.md:96]:
```suggestion
She is also a co-founder of the [CEVALE2VE project](http://www.cevale2ve.org/en/home/) that has become consolidated into [LA-CoNGA Physics](http://laconga.redclara.net/), an EU Erasmus+ funded project with a mission to create a Latin American and European Community for Advanced Physics.
```

**Comment by malvikasharan** [comment]
Amazing power women! Thank you @crangelsmith, @HeidiSeibold and @sgibson91 for such quick review. @HeidiSeibold I might have had a Freudian slip for tagging you by mistake but super glad that I did. 🎉 

With your feedback, I feel confident to unleash this.

**Comment by malvikasharan** [comment]
Thank you @KirstieJane! 🌸Merging it! 🥳️

**Pull request merged**